extensions/config/portability-config-yaml/src/main/java/org/dataportabilityproject/config/yaml/YamlSettingsExtension.java

```diff
@@ -43,6 +47,8 @@
   private static final String API_SETTINGS_PATH = "config/api.yaml";
   private static final String ENV_API_SETTINGS_PATH = "config/env/api.yaml";
 
+  private static final String RETRY_LIBRARY_PATH = "config/default_retry.yaml";
```

Comment:
can be a TODO for later, but it seems arbitrary that we'd require a file called default_retry.yaml and that's the only one we parse. I think it'd be better to read in config/retry/\*.yaml (to allow integrators to structure their retry policies however they want, including multiple files) and config/env/retry/\*.yaml (to allow for environment specific overrides of defaults, like we do for the other yaml settings).

Reply:
Good idea!  Doesn't look like our current tools support wildcard expansion, but we should do that.